### DISCUSSION TITLE
Shared Styles across multiple Apps in the same directory, Shared Custom Library.

### DISCUSSION THREAD

**Discussion by clojurelife** [AvaloniaUI/Avalonia#9102] [Q&A] [answered]
I have several Avalonia Apps in a projects directory and I would like to keep my styles, icons, etc, in my "stylesLibrary". How do i reference those styles from those app directories. Or do we have to we have to repeat code every time we create a new App.
projects/
projects/app1
projects/app2
projects/app3
projects/stylesLibrary

Any help would be great. I found these discusions, but i haven't been able to get my head rapped around it.
Shared DataTemplates across projects:
https://github.com/AvaloniaUI/Avalonia/discussions/8426

Resouces.xaml in Avalonia
https://stackoverflow.com/questions/57910252/resource-xaml-in-avalonia

**Comment by timunie** [comment, marked as answer]
I have my own lib with all my styles and reuseable control which I can include elsewhere. You can use StyleInclude with the URI pointing to this lib.

**Reply by timunie** [reply]
Have a look how Aura.UI works for example
